CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # OpenAPI-Postman Changelog
+#### v1.1.13 (April 21, 2020)
+* Added support for detailed validation body mismatches with option detailedBlobValidation.
+* Fixed unable to validate schema with type array. [#8098](https://github.com/postmanlabs/postman-app-support/issues/8098)
```

Comment:
For github issues, let's follow the same format as before - `Fix for <url> - <description>`